code/modules/urist/modules/wfc/_module.dm

```diff
@@ -0,0 +1,32 @@
+// Core module definition; similar to map module defines
+// If you use Dream Maker, everything is unticked b/c we include stuff from this file.
+
+// flag we can use to avoid double-including
+#define DEEPMAINT_INCLUDED 1
+
+#include "_wfc_defines.dm"
+
+#ifdef STUB_OUT_SS13
+
+#include "ss13_stubs.dm"
+#include "modules\deepmaint\areas.dm"
+
+#else
+
+#include "fractal_coords.dm"
+#include "modules\deepmaint\hooks.dm"
+
+#endif
+
+#include "core\calllib.dm"
+//#include "core\debug.dm"
```

Comment:
Could be an idea to put this behind some kind of debug `#ifdef`? Along with more verbose logging, like we did in the GOAI project?

Reply:
I think that's redundant in this case, if you want these you can just uncomment the line or manually include debug. It's mostly debug _verbs_ anyway.